docs/source/development/add-ons/test-add-ons-18.md

```diff
@@ -13,51 +13,259 @@ myst:
+For more details, refer to the [Vitest Mocking Guide](https://vitest.dev/guide/mocking.html).
+
+
+### Testing with lazy loaded libraries
+
+In Jest, you would test lazy loaded libraries as shown.
+
+```javascript
+jest.mock('@plone/volto/helpers/Loadable/Loadable');
+beforeAll(
+  async () =>
+    await require('@plone/volto/helpers/Loadable/Loadable').__setLoadables(),
+);
+```
+
+In Vitest the equivalent is the following example.
+
+```javascript
+vi.mock('@plone/volto/helpers/Loadable/Loadable');
+beforeAll(async () => {
+  const { __setLoadables } = await import('@plone/volto/helpers/Loadable/Loadable');
+  await __setLoadables();
+});
+```
+
+`require()` is not supported in Vitest.
+Instead, use `import()`.
+
+
+## Run tests in Vitest
+
+After completing the foregoing steps, you can use Vitest.
+To run your tests, execute the following command.
 
 ```shell
-make acceptance-frontend-dev-start
+make test
+```
+
+
+## `jiti` for add-on testing
+
+[`jiti`](https://www.npmjs.com/package/jiti) is a module loader for JavaScript and TypeScript that allows you to import and execute TypeScript files or ESM modules without requiring a build step.
+It is particularly useful in Volto for dynamically loading add-ons and their configurations.
+
+To use `jiti`, see the following code example.
+
+```javascript
+import jiti from 'jiti';
+
+const _import = jiti(import.meta.url, { esmResolve: true });
 ```
 
-You can run the frontend in development mode, so you can develop while writing tests.
-Run the following command to run Cypress tests afterward.
+In Vitest, `require()` is not supported, and `import()` may sometimes fail.
+In such cases, `jiti` provides a reliable alternative for module resolution and execution.
+In Volto, `jiti` is used in the test file {file}`packages/volto/__tests__/create-addons-loader.test.js` to handle dynamic imports for add-ons efficiently.
+
+
+## Additional Vitest information
+
+```{seealso}
+For a general guide for Vitest, see {doc}`../../contributing/testing`.
+```
+For complete details on migrating from Jest to Vitest, refer to the official [Vitest Migration Guide](https://vitest.dev/guide/migration.html#jest).
+
+
+## Jest for Volto add-ons
+
+Volto previously used {term}`Jest` as its unit testing framework.
+If your add-on still uses Jest, Volto provides backward compatibility.
+
+While Volto core has migrated to Vitest, it retained its Jest configuration, allowing you to continue running your existing Jest tests in your add-on without issues.
+
+This support lets you migrate your tests from Jest to Vitest at your convenience.
+
+
+### Jest configuration override
+
+If you use Jest for your add-on testing, you may need to customize the configuration, especially in GitHub workflows or local development.
+Volto provides a way to do this by either using a {file}`jest.config.js` file, or specifying a custom configuration file through the `RAZZLE_JEST_CONFIG` environment variable.
```

Comment:
I recently found the [`envvar` role in Sphinx](https://www.sphinx-doc.org/en/master/usage/referencing.html#role-envvar).

```suggestion
Volto provides a way to do this by either using a {file}`jest.config.js` file, or specifying a custom configuration file through the {envvar}`RAZZLE_JEST_CONFIG` environment variable.
```